plugin/jvm/src/main/java/com/twosigma/beaker/easyform/EasyForm.java

```diff
@@ -35,7 +35,7 @@
 import java.util.Map;
 
 @SuppressWarnings("unchecked")
-public class EasyForm extends ObservableMap<String, String> {
+public class EasyForm extends ObservableMap<String, Object> {
```

Comment:
Type of value is checked at compile time, so that exception is thrown `java.lang.ClassCastException: java.util.Date cannot be cast to java.lang.String`.
To accept different types that parameter should be Object.